JS-Algorithms/challenges.js

```diff
@@ -14,9 +14,22 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 
 const readableTime = (seconds) => {
   // YOUR CODE HERE...
+  if(seconds > 0) {
+    const hours = Math.floor(seconds / 3600);
+    const minutes = Math.floor(seconds % 3600 / 60);
+    const sec = Math.floor(seconds % 3600 % 60);
```

Comment:
I find whenever im tempted to name a variable a shorter version of an existing one, it means I need to be more descriptive with my naming. What are these `hours` `minutes` `seconds` variables really?